plugins/kotlin-as-java/src/main/kotlin/transformToJava.kt

```diff
@@ -7,23 +7,24 @@ import org.jetbrains.dokka.model.DFunction
 import org.jetbrains.dokka.model.DPackage
 import org.jetbrains.dokka.model.DProperty
 import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.utilities.DokkaLogger
 
 private val JVM_NAME_DOCUMENTABLE_TRANSFORMER by lazy {
     JvmNameDocumentableTransformer()
 }
 
-fun DPackage.transformToJava(context: DokkaContext): DPackage {
-    return JVM_NAME_DOCUMENTABLE_TRANSFORMER.transform(this.asJava(), context)
+fun DPackage.transformToJava(logger: DokkaLogger): DPackage {
+    return JVM_NAME_DOCUMENTABLE_TRANSFORMER.transform(this.asJava(), logger)
```

Comment:
Any particular reason for changing context to just logger? Since this is public API, maybe it's worth having `context` here? It'll take some deprication cycles to change this back to context in the future